### PR TITLE
add autofs md

### PR DESCRIPTION
**Pull request by fengshunli**
add doc https://github.com/cubefs/cubefs_blog/issues/17

### CODE REVIEW

**Comment by heymingwei** [comment]
the picture also need to be translated @fengshunli

**Comment by heymingwei** [comment]
the file name need to change to "cubefs_autofs_practice"

File: blog/news/cubefs_autofs_practice.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+## Autofs Configuration mounting example
+
+The configuration file for autofs is /etc/auto.master, which specifies the root directory for automatic mounts and the location of the configuration file. When we access a subdirectory under this root directory, autofs will automatically mount the corresponding file system based on the configuration file.
+
+### /etc/auto.master example:
+
+```bash
+/- /etc/auto.direct -ro,hard,intr,nolock 
+# 增加 /tmp/cfstest /etc/auto.cfs
```
Comment (heymingwei) [inline blog/news/cubefs_autofs_practice.md:72]: english

Comment (fengshunli) [reply blog/news/cubefs_autofs_practice.md]: done

**Comment by heymingwei** [comment]
I find it still need to update a file: blog/README.md，otherwise it can't be display. @fengshunli

**Comment by fengshunli** [comment]
done

**Review by heymingwei — APPROVED** [review]

**Comment by NaturalSelect** [comment]
@fengshunli Please resolve conflicts.

**Comment by fengshunli** [comment]
@NaturalSelect done

**Review by NaturalSelect — APPROVED** [review]
LGTM

**Pull request merged**